core/src/main/java/org/elasticsearch/rest/action/get/RestHeadAction.java

```diff
@@ -57,7 +57,7 @@ public void handleRequest(final RestRequest request, final RestChannel channel,
         getRequest.routing(request.param("routing"));  // order is important, set it after routing, so it will set the routing
         getRequest.parent(request.param("parent"));
         getRequest.preference(request.param("preference"));
-        getRequest.realtime(request.paramAsBoolean("realtime", null));
+        getRequest.realtime(request.paramAsBoolean("realtime", true));
```

Comment:
I think it is more normal to do `getRequest.realtime(request.paramAsBoolean("realtime", getRequest.realtim());`. Just so the default is on the request rather than in the API.